setup.cfg

```diff
@@ -28,7 +28,7 @@ install_requires =
 [options.extras_require]
 test =
     numpy
-    astropy
+    astropy>=4.0
```

Comment:
astropy is an optional requirement but the plugin does nothing if astropy is not installed. Is it to avoid a cross dependency which would confuse pip ?

Reply:
I noticed that too! I just thought since `astropy` is in the plugin name, it is not unreasonable to require `astropy` to be installed, but it did still catch me by surprise.

Reply:
Let's address this part in a separate PR or issue.